bin/lt.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+
+const openurl = require('openurl');
+const yargs = require('yargs');
+
+const localtunnel = require('../localtunnel');
+const { version } = require('../package');
+
+const { argv } = yargs
+  .usage('Usage: lt --port [num] <options>')
+  .env(true)
+  .option('h', {
+    alias: 'host',
+    describe: 'Upstream server providing forwarding',
+    default: 'https://localtunnel.me',
+  })
+  .option('s', {
+    alias: 'subdomain',
+    describe: 'Request this subdomain',
+  })
+  .option('l', {
+    alias: 'local-host',
+    describe: 'Tunnel traffic to this host instead of localhost, override Host header to this host',
+  })
+  .options('o', {
+    alias: 'open',
+    describe: 'Opens url in your browser',
+  })
+  .option('p', {
+    alias: 'port',
+    describe: 'Internal http server port',
+  })
+  .option('print-requests', {
+    describe: 'Print basic request info',
+  })
+  .require('port')
+  .boolean('print-requests')
+  .help('help', 'Show this help and exit')
+  .version(version);
```

Comment:
Should the local key and cert options be exposed here? Along with an option to allow invalid certs?

Reply:
Done.